ledger/src/blockstore_options.rs

```diff
@@ -155,24 +155,18 @@ pub struct BlockstoreRocksFifoOptions {
     pub shred_code_cf_size: u64,
 }
 
-// Maximum size of cf::DataShred.  Used when `shred_storage_type`
-// is set to ShredStorageType::RocksFifo.  The default value is set
-// to 125GB, assuming 500GB total storage for ledger and 25% is
-// used by data shreds.
-const DEFAULT_FIFO_COMPACTION_DATA_CF_SIZE: u64 = 125 * 1024 * 1024 * 1024;
-// Maximum size of cf::CodeShred.  Used when `shred_storage_type`
-// is set to ShredStorageType::RocksFifo.  The default value is set
-// to 100GB, assuming 500GB total storage for ledger and 20% is
-// used by coding shreds.
-const DEFAULT_FIFO_COMPACTION_CODING_CF_SIZE: u64 = 100 * 1024 * 1024 * 1024;
+// The default storage size for storing shreds when `rocksdb-shred-compaction`
+// is set to `fifo` in the validator arguments.  This amount of storage size
+// in bytes will equally allocated to both data shreds and coding shreds.
+pub const DEFAULT_ROCKS_FIFO_SHRED_STORAGE_SIZE_BYTES: u64 = 250 * 1024 * 1024 * 1024;
```

Comment:
It is nice that this is now one constant instead of two; however, I still don't love that we have the divide-by-two in multiple places (here and validator/src/main.rs). The concern is that if we tune the ratio in one spot, we have to remember to tune it in the other.

Why not just remove the default trait implementation on `BlockstoreRocksFifoOptions` since the only caller of this specifies the fields anyways?

Reply:
> It is nice that this is now one constant instead of two; however, I still don't love that we have the divide-by-two in multiple places (here and validator/src/main.rs). The concern is that if we tune the ratio in one spot, we have to remember to tune it in the other.

This is addressed in the original PR #25569 where it introduces a helper function that takes the total shred storage size and allocates (currently 1:1) sizes for both data and coding shred.